onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -578,7 +578,13 @@ static void RegisterExecutionProviders(InferenceSession* sess, const std::vector
       auto it = provider_options_map.find(type);
       if (it != provider_options_map.end()) {
         for (auto option : it->second) {
-          if (option.first == "has_trt_options") {
+          if (option.first == "device_id") {
```

Comment:
we need python unit tests for these as I mentioned in the original PR where these were added. they can follow the CUDA EP example in onnxruntime_test_python.py

Reply:
sure. will do